src/jit/compiler.h

```diff
@@ -1392,7 +1393,7 @@ struct fgArgTabEntry
 
 #if defined(FEATURE_UNIX_AMD64_STRUCT_PASSING)
     regNumber             otherRegNum;              // The (second) register to use when passing this argument.
-    bool                  isStruct;                 // is this a struct arg
+    bool           isStruct     :1; // True if this is a struct arg
```

Comment:
Should you move this up, so it can get allocated next to the other bool bitfields?